### PR TITLE
FI-42-JOIN-command

### PR DESCRIPTION
**Pull request by ynslee**

### CODE REVIEW

File: includes/Reply.hpp [hunk 1]
```diff
@@ -2,6 +2,8 @@
 #ifndef REPLY_HPP
 # define REPLY_HPP
 
+# define user(nick, username, userIP)(":" + nick + "!" + username + "@" + userIP)
```
Comment (hussojo) [inline includes/Reply.hpp:5]: Should we write the macro "user" also on caps? -> USER

**Review by jestebanpelaez18 — APPROVED** [review]
Looks good ! will try too test it when i finnish quit command

File: srcs/commands/Join.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+			}
+		}
+	}
+	channels[channelName] = new Channel(channelName);
+	channels[channelName]->addToChannel(*Client);
+	channels[channelName]->setChannelKey(key);
+	return (0);
+}
+
+int cmdJoin(Message &msg, Client *Client, std::map<std::string, Channel*> &channels)
```
Comment (hussojo) [inline srcs/commands/Join.cpp:59]: Do we need an error in case of channel name is already in use?

Comment (ynslee) [reply srcs/commands/Join.cpp:59]: It is only partly ready so those will be added later. I am pushing it so Juan can test quit commands with one channel :)

File: includes/Reply.hpp (hunk 1 above)
Comment (ynslee) [reply includes/Reply.hpp:5]: yup. will change it now

**Pull request merged**